src/Microsoft.AspNet.Razor/CodeGenerators/GeneratedTagHelperContext.cs

```diff
@@ -186,24 +187,29 @@ public GeneratedTagHelperContext()
         public string TagHelperContentTypeName { get; set; }
 
         /// <summary>
-        /// The name of the method used to write <see cref="ExecutionContextTypeName"/>.
+        /// The name of the property containing the <c>HtmlEncoder</c>.
         /// </summary>
-        public string WriteTagHelperAsyncMethodName { get; set; }
+        public string HtmlEncoderPropertyName { get; set; }
 
         /// <summary>
-        /// The name of the method used to write <see cref="ExecutionContextTypeName"/> to a specified
-        /// <see cref="System.IO.TextWriter"/>.
+        /// The name of the method used to convert a <c>TagHelperContent</c> into a <see cref="string"/>.
         /// </summary>
-        public string WriteTagHelperToAsyncMethodName { get; set; }
+        public string TagHelperContentGetContentMethodName { get; set; }
 
         /// <summary>
-        /// The name of the property containing the <c>HtmlEncoder</c>.
+        /// The property accessor to indicate if the tag helper's content has been modified.
```

Comment:
Ick. This is still a property name and that wording is consistent with the rest of this class.